### ISSUE TITLE
Current build (3.6.0) unimportable

### ISSUE THREAD

**Issue by schiroo-wiw** [HubSpot/hubspot-api-python#29]
After installing the latest version of the hubspot api client (3.6.0) the `hubspot` package can no longer be imported. 

Reproduced here within the current `python:latest` docker image:

```
root@b24251133b8c:/# pip3 install hubspot-api-client
Collecting hubspot-api-client
  Downloading hubspot_api_client-3.6.0-py3-none-any.whl (1.7 MB)
     |████████████████████████████████| 1.7 MB 4.3 MB/s
Collecting python-dateutil
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 12.6 MB/s
Collecting certifi
  Downloading certifi-2020.12.5-py2.py3-none-any.whl (147 kB)
     |████████████████████████████████| 147 kB 7.3 MB/s
Collecting urllib3>=1.15
  Downloading urllib3-1.26.3-py2.py3-none-any.whl (137 kB)
     |████████████████████████████████| 137 kB 9.3 MB/s
Collecting six>=1.10
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Installing collected packages: six, urllib3, python-dateutil, certifi, hubspot-api-client
Successfully installed certifi-2020.12.5 hubspot-api-client-3.6.0 python-dateutil-2.8.1 six-1.15.0 urllib3-1.26.3
root@b24251133b8c:/# python3
Python 3.9.1 (default, Jan 12 2021, 16:45:25)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import hubspot
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.9/site-packages/hubspot/__init__.py", line 1, in <module>
    from .client import Client
  File "/usr/local/lib/python3.9/site-packages/hubspot/client.py", line 6, in <module>
    from .discovery.files.discovery import Discovery as FilesDiscovery
  File "/usr/local/lib/python3.9/site-packages/hubspot/discovery/files/discovery.py", line 2, in <module>
    from .files.discovery import Discovery as FilesDiscovery
  File "/usr/local/lib/python3.9/site-packages/hubspot/discovery/files/files/discovery.py", line 1, in <module>
    import hubspot.files.files as api_client
ModuleNotFoundError: No module named 'hubspot.files'
>>>
```

**Comment by schiroo-wiw** [comment]
Digging a bit more into this, I'd speculate the issue is that `hubspot.files` isn't a package, since `hubspot/files/` doesn't contain an `__init__.py` (`hubspot/files/files/` does, but I don't think that counts). It looks like this results in the whole directory getting left out of the build(at least when I dig around in my `site-packages`), which may explain why it wasn't caught by a test.

**Comment by atanasiuk-hubspot** [comment]
Hi, @schiroo-wiw, Thank you for reporting this! This is indeed because of missing `__init__.py`. Fixed in 3.6.1

**Comment by schiroo-wiw** [comment]
Awesome, thanks! 🏆
